Hunting Queries/OracleDatabaseAudit/OracleDBAuditActionsByUser.yaml

```diff
@@ -0,0 +1,30 @@
+id: e0b49a22-1c63-4572-b913-b3f31b8e9edf
+name: OracleDBAudit - Action by user
+description: |
+  'Query searches actions made by user.'
+requiredDataConnectors:
+  - connectorId: OracleDatabaseAudit
+    dataTypes:
+      - Syslog
+tactics:
+  - InitialAccess
+  - DefenseEvasion
+  - Collection
+  - Impact
+query: |
+  let lbtime = 24h;
+  OracleDatabaseAuditEvent
+  | where TimeGenerated > ago(lbtime)
+  | where isnotempty(DbAction)
+  | where isnotempty(DstUserName)
+  | where DbAction in~ ('SELECT', 'DROP', 'INSERT')
+  | extend TableName = replace(@'[,\(\)]', '', extract(@'(?i)SELECT(.*?)FROM\s(.*?)\s', 2, Action))
+  | where isnotempty(TableName)
+  | where TableName !in ('SELECT', 'select')

```

Comment:
use `| where TableName !~ 'select'` <-- case insensitive match

Reply:
updated

Reply:
looks like this one did not get updated

Reply:
fixed